README.md

```diff
@@ -57,9 +57,6 @@ all dependencies for PDF.js:
 
     $ npm install
 
-> [!NOTE]
-> On MacOS M1/M2 you may see some `node-gyp`-related errors when running `npm install`. This is because one of our dependencies, `"canvas"`, does not provide pre-built binaries for this platform and instead `npm` will try to build it from source. Please make sure to first install the necessary native dependencies using `brew`: https://github.com/Automattic/node-canvas#compiling.
```

Comment:
Does the new package "just work" on MacOS? :)

Reply:
Yes, it at least *installs* successfully there as shown in a modified CI run that I triggered previously: https://github.com/mozilla/pdf.js/actions/runs/11744596145

Reply:
Note that `node-gyp` is not used by this new package, see the "About" section at https://github.com/Brooooooklyn/canvas, which is probably why this is not an issue anymore.